app-network/ucx/autobuild/defines

```diff
@@ -0,0 +1,24 @@
+PKGNAME=ucx
+PKGSEC=science
```

Comment:
Probably a `libs`?

Reply:
The package is placed under `app-network` by mistake. I'll adjust its location.